.github/workflows/findbugs.yml

```diff
@@ -20,6 +20,13 @@ jobs:
       with:
         egress-policy: audit
 
+    - name: Install Python
+      uses: actions/setup-python@65d7f2d534ac1bc67fcd62888c5f4f3d2cb2b236
+      with:
+        python-version: '3.10'
+    - name: Install scanoss-py
+      run: pip3 install scanoss
+
```

Comment:
Is installing python and scanoss-py necessary to the findbugs workflow?

Reply:
Agree. I've removed this from PR.